admin-dev/themes/new-theme/js/pages/product/components/filters/Filters.vue

```diff
@@ -126,7 +130,6 @@
       },
       clearAll(): void {
         this.selectedFilters = [];
-        this.$emit('clearAll');
```

Comment:
You can also clear `this.eventEmitter.emit(CombinationEvents.clearAllCombinationFilters);` No one listens to this event anymore

Reply:
So you should also be able to clean `clearAllCombinationFilters` from the event map